AtReconstruction/AtFilter/AtSCACorrect.cxx

```diff
@@ -61,33 +58,50 @@ AtPad *AtSCACorrect::getMatchingPad(AtPad *pad, AtPadReference *padReference, At
 void AtSCACorrect::removeBaseline(AtPad *pad, AtPadReference *padReference)
 {
 
-   AtPad *baselinePad = getMatchingPad(pad, padReference, fBaseline.get());
+   auto padRef = new AtPadReference(*padReference);
```

Comment:
This is a memory leak.

Do `auto padRef = *padRefenrece` or something similar.